tests/ChatTests/Groups.hs

```diff
@@ -3023,8 +3034,301 @@ testGLinkManualAcceptMember =
   where
     cfg = testCfg {chatHooks = defaultChatHooks {acceptMember = Just (\_ _ _ -> pure $ Right (GAPending, GRObserver))}}
 
-testGLinkDeletePendingMember :: HasCallStack => TestParams -> IO ()
-testGLinkDeletePendingMember =
+testGLinkReviewMember :: HasCallStack => TestParams -> IO ()
+testGLinkReviewMember =
+  testChat5 aliceProfile bobProfile cathProfile danProfile eveProfile $
+    \alice bob cath dan eve -> do
+      createGroup4 "team" alice (bob, GRMember) (cath, GRModerator) (dan, GRModerator)
+
+      alice ##> "/set new member review #team on"
+      alice <## "updated group preferences:"
+      alice <## "New member review: on"
+      concurrentlyN_
+        [ do
+            bob <## "alice updated group #team:"
+            bob <## "updated group preferences:"
+            bob <## "New member review: on",
+          do
+            cath <## "alice updated group #team:"
+            cath <## "updated group preferences:"
+            cath <## "New member review: on",
+          do
+            dan <## "alice updated group #team:"
+            dan <## "updated group preferences:"
+            dan <## "New member review: on"
+        ]
+
+      alice ##> "/create link #team"
+      gLink <- getGroupLink alice "team" GRMember True
+      eve ##> ("/c " <> gLink)
+      eve <## "connection request sent!"
+      alice <## "eve (Eve): accepting request to join group #team..."
+      concurrentlyN_
+        [ alice <## "#team: eve connected and pending review",
+          eve
+            <### [ "#team: joining the group...",
+                   "#team: you joined the group, pending review",
```

Comment:
We need some other language here, something like this:

#team: connecting to group moderators for admission to group